### PR TITLE
Add backstage weekly 29

### PR DESCRIPTION
**Pull request by dtuite**

### CODE REVIEW

**Comment by martina-if** [comment]
In the last section, would it make sense to add an example of how the `Parameters.dryRun` would be enabled?

File: content/blog/backstage-weekly-29-search-v1-alpha/index.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+### Search V1 Alpha released
+
+A new version of Backstage search was merged last week. Here's how the new search page looks in a Backstage example app:
+
+![a searchbar and a list of services that can be searched](./search-default-experience.png)
+
+This release is the culmination of a lot of hard work by the search team in Spotify and includes the following features:
+
+- A highly extensible search backend, responsible for document indexing and query handling. Previously, all search happened within the browser.
+- An out-of-the-box server-side search engine (Lunr), with an API in place to support other search engines in the future. ElasticSearch support is planned.
```
Comment (Xantier) [inline content/blog/backstage-weekly-29-search-v1-alpha/index.md:25]: Is this correct and they are using lunr on node.js. The tagline for lunr is:
```

Simple

Designed to be small, yet full featured, Lunr enables you to provide a great search experience **without the need for external, server-side,** search services.
```

**Review by Xantier — DISMISSED** [review]

File: content/blog/backstage-weekly-29-search-v1-alpha/index.md (hunk 1 above)
Comment (dtuite) [reply content/blog/backstage-weekly-29-search-v1-alpha/index.md:25]: @Xantier Here's the PR I got this from: https://github.com/backstage/backstage/pull/5927

They say "A no-config-needed, out-of-the-box, in-memory search engine (with an API in place to support other search engines)"

My understanding is that Lunr is an in-memory search library (aka. "engine") which runs in the Backstage backend.

So what I said is correct... "An out-of-the-box server-side search engine (Lunr),"

Does that sound correct to you?

**Comment by dtuite** [comment]
> In the last section, would it make sense to add an example of how the `Parameters.dryRun` would be enabled?

@martina-if I have added an explanation of how the "dryRun" parameter is rendered in the UI. I'm a bit too lazy to fire up Backstage again to get another screenshot. 😄

File: content/blog/backstage-weekly-29-search-v1-alpha/index.md (hunk 1 above)
Comment (Xantier) [reply content/blog/backstage-weekly-29-search-v1-alpha/index.md:25]: aight, looks grand

**Review by Irma12 — APPROVED** [review]

**Review by martina-if — APPROVED** [review]

**Pull request merged**